pkgs/by-name/ca/cargo-aoc/package.nix

```
@@ -0,0 +1,29 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-aoc";
+  version = "0.3.7";
+
+  src = fetchFromGitHub {
+    owner = "gobanos";
+    repo = "cargo-aoc";
+    rev = version;
```

```suggestion
    tag = version;
```
Nitpick. See https://github.com/NixOS/nixpkgs/pull/355973